### ISSUE TITLE
Error in Jupyter notebook tutorial

### ISSUE THREAD

**Issue by liangcg** [Krastanov/QuantumClifford.jl#8]
In the code block

```julia
initial_state = good_bell_state⊗good_bell_state
circuit = [
    SparseGate(CPHASE, [1,3]),
    SparseGate(CPHASE, [2,4]),
    BellMeasurement([X,X], [3,4]),
    VerifyOp(good_bell_state, [1,2])
]
displaycircuit(circuit)
```

The following error occurs:

MethodError: no method matching length(::SparseGate)

Stacktrace:
 [1] _similar_for(::UnitRange{Int64}, ::Type{Any}, ::SparseGate, ::Base.HasLength) at .\array.jl:597
 [2] _collect(::UnitRange{Int64}, ::SparseGate, ::Base.HasEltype, ::Base.HasLength) at .\array.jl:630
 [3] collect(::SparseGate) at .\array.jl:624
 [4] broadcastable(::SparseGate) at .\broadcast.jl:682
 [5] broadcasted at .\broadcast.jl:1255 [inlined]
 [6] circuitwidth(::SparseGate) at C:\Users\lcg75\.julia\packages\Quantikz\Ni7Ey\src\Quantikz.jl:321
 [7] _broadcast_getindex_evalf at .\broadcast.jl:648 [inlined]
 [8] _broadcast_getindex at .\broadcast.jl:621 [inlined]
 [9] getindex at .\broadcast.jl:575 [inlined]
 [10] copy at .\broadcast.jl:876 [inlined]
 [11] materialize at .\broadcast.jl:837 [inlined]
 [12] circuitwidth(::Array{AbstractOperation,1}) at C:\Users\lcg75\.julia\packages\Quantikz\Ni7Ey\src\Quantikz.jl:321
 [13] displaycircuit(::Array{AbstractOperation,1}; scale::Int64, kw::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at C:\Users\lcg75\.julia\packages\Quantikz\Ni7Ey\src\Quantikz.jl:449
 [14] displaycircuit(::Array{AbstractOperation,1}) at C:\Users\lcg75\.julia\packages\Quantikz\Ni7Ey\src\Quantikz.jl:449
 [15] top-level scope at In[38]:8
 [16] include_string(::Function, ::Module, ::String, ::String) at .\loading.jl:1091

**Comment by Krastanov** [comment]
Thanks for reporting that. It is due to a bug in `Quantikz`, the circuit drawing library. I will release a fix in the next few hours. In the meantime, simply add a semicolon to the end of your cell. That will turn off the cell output drawing (only the circuit drawing is broken, the rest should work, so you can still make a circuit and test it).

**Comment by Krastanov** [comment]
Should be fixed now (version 0.2.6). The notebooks were also slightly updated. It should install Quantikz 1.0.6. Updating should be as easy as `] update QuantumClifford`
